backend/FacilityBooking/FacilitiesAPI.py

```diff
@@ -163,14 +163,22 @@ def user_bookings(userID):
 @ cross_origin(supports_credentials=True)
 def check_bookings(facilityID):
     try:
+        condition = {
+            "$and": [
+                {'facilityID': int(facilityID)},
+            ]
+        }
+        if (request.args.get('endTime') != None):
+            condition['$and'].append({'startTime': {'$lte': int(
+                request.args.get('endTime'))}})
+
+        if (request.args.get('startTime') != None):
+            condition['$and'].append({'endTime': {'$gte': int(
```

Comment:
could I request that this change from `gte` to `gt`? This is so that bookings ending on midnight does not show up for the next day's bookings